core/lib/dal/src/events_dal.rs

```diff
@@ -222,7 +222,8 @@ impl EventsDal<'_, '_> {
                 tx_hash,
                 tx_index_in_block,
                 event_index_in_block,
-                event_index_in_tx
+                event_index_in_tx,
+                NULL::BIGINT AS "block_timestamp?"
```

Comment:
Fields that are set to null here (block_hash, l1_batch_number) are filled in `get_transaction_receipts`. You should fill block_timestamp as well